optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/SingleStatistic.java

```diff
@@ -178,6 +178,18 @@ public void readCsvStatisticFile() {
         }
     }
 
+    public void retrievePointList() {
+        if (getCsvFile().exists() && pointList == null) {
```

Comment:
This PR is kind-of blocked by https://issues.jboss.org/browse/PLANNER-427 and #137. I don't want to merge this if I'm not sure it works, that's why I didn't push the fixes yet.